chapter2/tests/t20/terraform/dev/ec2.tf

```diff
@@ -0,0 +1,37 @@
+module "ec2_instances" {
+  source        = "terraform-aws-modules/ec2-instance/aws"
+  count         = var.ec2_instances_count
+  name          = "${local.labels.ec2_instance}-${count.index + 1}"
```

Comment:
use null label module with proper attributes

Reply:
> use null label module with proper attributes

@dmitry-mightydevops 
Please give a comment, I didn't understand what was meant

Reply:
Not done, you just moved this from ec2.tf to locals.tf